### ISSUE TITLE
How to decouple dedicate index storage by project?

### ISSUE THREAD

**Issue by kristoflarcher** [opensearch-project/OpenSearch-Dashboards#4809]
**Versions** (relevant - OpenSearch/Dashboard/Server OS/Browser): 
2.2.0, running on VMs

**Describe the issue**: 
Hi ,
I push all applicative logs on 3 Opensearch nodes 
Everything is working perfectly fine for 1 year now.

We would need to push logs from our APIC Tools : but its 10 X more  logs than the applicative logs.

We would like to be able to make these logs available in the same Opensearch Instance
But without compromising our applicative indexes.

We would like to set the shards routing properly to fullfill our needs
And we wonder how we can implement it properly.

In other words we wonder how we can set Opensearch Worker nodes to store exclusively 
 - applicative shards ( master / replicas ) 
 - APIC  shards  ( master  / replicas )

Is is possible to decouple Worker nodes 
  -  some nodes would be dedicated to Applicative shards storing
  -  the others nodes would be dedicated  to  APIC shards storing

Thank you for support

**Comment by ashwin-pc** [comment]
@opensearch-project/admin Can you move this to the core repo?
